packages/block-library/src/gallery/style.scss

```diff
@@ -1,8 +1,10 @@
-.wp-block-gallery {
+.wp-block-gallery,
+.blocks-gallery-grid {
```

Comment:
This seems a bit strange to me as it will duplicate all the nested rules, even the ones targeting child elements. Could it be separated with only the necessary rules added?

Reply:
I was concerned removing all the styles from `wp-block-gallery` might break deprecated instances of the block. If that's not a problem I can get rid of it.